hypothesis-python/tests/common/debug.py

```diff
@@ -38,7 +39,7 @@ def minimal(
     settings = Settings(
         settings,
         max_examples=50000,
-        database=None,
+        database=None, timeout=unlimited,
```

Comment:
style bit: shouldn't `timeout` argument be on the next line?

Reply:
If anything there are too many lines there. I've collapsed it down onto one.